### ISSUE TITLE
Error when installing via composer

### ISSUE THREAD

**Issue by FatBoyXPC** [pionl/laravel-chunk-upload#5]
I am attempting to clone an existing Laravel application and get it running locally. When I run `composer install` it has the following error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for pion/laravel-chunk-upload ^0.2.3 -> satisfiable by pion/laravel-chunk-upload[v0.2.3].
    - Conclusion: don't install laravel/framework v5.2.45
    - Conclusion: don't install laravel/framework v5.2.44
    - Conclusion: don't install laravel/framework v5.2.43
    - Conclusion: don't install laravel/framework v5.2.42
    - Conclusion: don't install laravel/framework 5.2.41
    - Conclusion: don't install laravel/framework v5.2.40
    - Conclusion: don't install laravel/framework v5.2.39
    - Conclusion: don't install laravel/framework v5.2.38
    - Conclusion: don't install laravel/framework v5.2.37
    - Conclusion: don't install laravel/framework v5.2.36
    - Conclusion: don't install laravel/framework v5.2.35
    - Conclusion: don't install laravel/framework v5.2.34
    - Conclusion: don't install laravel/framework v5.2.33
    - Conclusion: don't install laravel/framework v5.2.32
    - Conclusion: don't install laravel/framework v5.2.31
    - Conclusion: don't install laravel/framework v5.2.30
    - Conclusion: don't install laravel/framework v5.2.29
    - Conclusion: don't install laravel/framework v5.2.28
    - Conclusion: don't install laravel/framework v5.2.27
    - Conclusion: don't install laravel/framework v5.2.26
    - Conclusion: don't install laravel/framework v5.2.25
    - Conclusion: don't install laravel/framework v5.2.24
    - Conclusion: don't install laravel/framework v5.2.23
    - Conclusion: don't install laravel/framework v5.2.22
    - Conclusion: don't install laravel/framework v5.2.21
    - Conclusion: don't install laravel/framework v5.2.20
    - Conclusion: don't install laravel/framework v5.2.19
    - Conclusion: don't install laravel/framework v5.2.18
    - Conclusion: don't install laravel/framework v5.2.17
    - Conclusion: don't install laravel/framework v5.2.16
    - Conclusion: don't install laravel/framework v5.2.15
    - Conclusion: don't install laravel/framework v5.2.14
    - Conclusion: don't install laravel/framework v5.2.13
    - Conclusion: don't install laravel/framework v5.2.12
    - Conclusion: don't install laravel/framework v5.2.11
    - Conclusion: don't install laravel/framework v5.2.10
    - Conclusion: don't install laravel/framework v5.2.9
    - Conclusion: don't install laravel/framework v5.2.8
    - Conclusion: don't install laravel/framework v5.2.7
    - Conclusion: don't install laravel/framework v5.2.6
    - Conclusion: don't install laravel/framework v5.2.5
    - Conclusion: don't install laravel/framework v5.2.4
    - Conclusion: don't install laravel/framework v5.2.3
```

The relevant line from `composer.json`:
```
        "pion/laravel-chunk-upload": "^0.2.3",
```

**Comment by pionl** [comment]
@FatBoyXPC Hi, what version of Laravel are you using?

**Comment by pionl** [comment]
@FatBoyXPC At first, I would recommend to use the latest version 0.3.2, it will include fixed schedule setup + specific Laravel versions (5.1.* || 5.2.* || 5.3.*).

I've installed the package on 5.1 and 5.2 and 5.3 without problem.

**Comment by FatBoyXPC** [comment]
Interestingly enough, I removed a few unrelated packages from composer.json to get everything working. I'm not sure why or how that works, but I didn't complain as I needed to get stuff going.

If for some reason this package was not cooperating with the others:
Composer depends on 5.2.*, but the specific version composer installed was 5.2.45 once I finally got the other packages removed and composer actually installed.
